lib/carbon/tests/test_client.py

```diff
@@ -72,7 +72,7 @@ def setUp(self):
 
   def test_send_datapoints_now(self):
     datapoint = ('foo.bar', (1000000000, 1.0))
-    expected_line_to_send = "foo.bar 1.0 1000000000"
+    expected_line_to_send = "foo.bar 1.000000 1000000000"
```

Comment:
This is going to have a negative impact on bandwidth usage, we should be sending the smallest possible representation.

Reply:
I can't find anything smarter than `('%f' % x).rstrip('0').rstrip('.')`, any idea ?